ebt/Dialog/BtReciever.vala

```diff
@@ -203,36 +209,37 @@ public class BtReciever : Granite.Dialog {
             return;
         }
         if (elapsed_time == 0) {
-		    return;
-		}
-		uint64 transfer_rate = transferred / elapsed_time;
-		if (transfer_rate == 0) {
-		    return;
-		}
-		rate_label.label = _("<b>Rate:</b> %s").printf (GLib.format_size (transfer_rate));
-		uint64 remaining_time = (total_size - transferred) / transfer_rate;
-		progress_label.label = _("Receiving…  %s of %s remaining %s").printf (GLib.format_size (transferred), GLib.format_size (total_size), format_time((int)remaining_time));
+            return;
+        }
+        uint64 transfer_rate = transferred / elapsed_time;
+        if (transfer_rate == 0) {
+            return;
+        }
+        rate_label.label = _("<b>Transfer rate:</b> %s").printf (GLib.format_size (transfer_rate));
+        uint64 remaining_time = (total_size - transferred) / transfer_rate;
+        progress_label.label = _("Receiving… %s of Size: %s, remaining time %s").printf (GLib.format_size (transferred), GLib.format_size (total_size), format_time ((int)remaining_time));
         }
 
     private string format_time (int seconds) {
-	    int minutes;
-	    if (seconds < 0) {
-		    seconds = 0;
-		}
-	    if (seconds < 60) {
-		    return ngettext("%'d second", "%'d seconds", seconds).printf (seconds);
-		}
-	    if (seconds < 60 * 60) {
-		    minutes = (seconds + 30) / 60;
-		    return ngettext("%'d minute", "%'d minutes", minutes).printf (minutes);
-	    }
-	    int hours = seconds / (60 * 60);
-	    if (seconds < 60 * 60 * 4) {
-		    minutes = (seconds - hours * 60 * 60 + 30) / 60;
-		    string h = ngettext("%'u hour", "%'u hours", hours).printf (hours);
-		    string m = ngettext("%'u minute", "%'u minutes", minutes).printf (minutes);
-		    return h.concat(", ", m);
-	    }
-	    return ngettext("approximately %'d hour", "approximately %'d hours", hours).printf (hours);
+        int minutes;
+        if (seconds < 0) {
+            seconds = 0;
+        }
+        if (seconds < 60) {
```

Comment:
Elementary code style is to leave a blank line after an if clause.

Reply:
Ok